partials/java-class

```diff
@@ -1,15 +1,18 @@
-{%- macro javaClass(name, schema, properties, required, indentLevel, isStatic) %}
```

Comment:
What happened to required? Was it dead code?

Reply:
Yes, I never used the 'required' parameter.